DFC.Digital/DFC.Digital.Services.SendGrid/Services/ContactUsRequestMergeEmailContentService.cs

```diff
@@ -39,7 +40,7 @@ private static string TokenReplacement(ContactUsRequest sendEmailRequest, string
                 mergedContent = mergedContent.Replace(DobToken, sendEmailRequest.DateOfBirth?.ToString("dd/MM/yyyy"));
                 mergedContent = mergedContent.Replace(PostCodeToken, sendEmailRequest.Postcode);
                 mergedContent = mergedContent.Replace(ContactAdvisorQuestionTypeToken, sendEmailRequest.ContactAdviserQuestionType);
-                mergedContent = mergedContent.Replace(MessageToken, sendEmailRequest.Message);
+                mergedContent = mergedContent.Replace(MessageToken, sendEmailRequest.Message.Replace("\r\n", Environment.NewLine));
```

Comment:
@vallabha24 as discussed please include both variations we will need to replace

Reply:
I updated it and committed my changes